python/hartree_fock/impurity.py

```diff
@@ -38,6 +39,12 @@ class ImpuritySolver(object):
 
     beta : float
         inverse temperature
+    
+    U : float
```

Comment:
It is a bit inconsistent now that the user hands over U / J and h_int in the solve command. Can't we name it directly dc_U / dc_J, and make it also optional?